_data/team-members.yml

```diff
@@ -87,11 +87,11 @@ members:
     twitter: mnduhem
     photo: /resources/img/martinduhem.png
     currentlyWorkingOn: true
-    contact_for: [scalaNative, buildTools]
+    contact_for: [scalaNative, build-tools]
```

Comment:
Have you built this locally and made sure that these changes didn't break anything??